### PR TITLE
{data}[intel/2020b] update PnetCDF with newer intel 2020b toolchain

### PR DESCRIPTION
**Pull request by rarensu**
Update to newer intel toolchain. Very easy. Needed for CESM-deps update to same toolchain.

### CODE REVIEW

**Comment by rarensu** [comment]
Don't forget to look at PR #13017 after merging this one.

File: easybuild/easyconfigs/p/PnetCDF/PnetCDF-1.12.1-iimpi-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+easyblock = 'ConfigureMake'
+
+name = 'PnetCDF'
+version = "1.12.1"
```
Comment (lexming) [inline easybuild/easyconfigs/p/PnetCDF/PnetCDF-1.12.1-iimpi-2020b.eb:4]: Version 1.12.2 is available. Please bump this easyconfig to this last release,

**Review by lexming — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/p/PnetCDF/PnetCDF-1.12.1-iimpi-2020b.eb (hunk 1 above)
Comment (verdurin) [reply easybuild/easyconfigs/p/PnetCDF/PnetCDF-1.12.1-iimpi-2020b.eb:4]: @rarensu Any update on this one?

**Comment by boegel** [comment]
closing this since `intel/2020b` is deprecated, see https://docs.easybuild.io/policies/toolchains

**Pull request closed without merging**